package-examples/pipeline-validate/README.md

```diff
@@ -74,8 +74,8 @@ Render the changes in the rendering pipeline by using `kpt fn render` command:
 ```shell
 $ kpt fn render pipeline-validate/
 
-package "pipeline-validate": running function "gcr.io/kpt-fn/set-labels:unstable": SUCCESS
-package "pipeline-validate": running function "gcr.io/kpt-fn/enforce-gatekeeper:unstable": SUCCESS
+package "pipeline-validate": running function "set-labels:unstable": SUCCESS
```

Comment:
1. this output is old, and has changed. 
2. Even then, you want to drop the container registry prefix in the CL output. This PR should only affect the user input.

Reply:
Good catch on 2), yes output should print the fully resolved path so that users know what is getting executed (look at my previous comment, that should help).

Reply:
Updated the results. They will have full path.